tensorrt_llm/_torch/autotuner.py

```
@@ -733,6 +754,31 @@
         return tensors
 
+    def _prepare_input_tensors_with_batches(
+        self,
+        inputs: List[torch.Tensor],
+    ) -> Tuple[List[List[torch.Tensor]], int]:
+        # TODO: only consider tensor parameter?
+        one_buffer_bytes = sum(
+            input.numel() *
+            input.element_size() if isinstance(input, torch.Tensor) else 0
+            for input in inputs)
+        num_buffers = ceil(self._get_l2_cache_size_in_bytes() /
```

Ruff (0.12.2)

Undefined name `ceil`

(F821)